### PR TITLE
pyMANGA as pip package:

### PR DESCRIPTION
**Pull request by jvollhueter**
- 260821
- first commit

### CODE REVIEW

File: MANGA.py [hunk 1]
```diff
@@ -8,7 +8,37 @@
 from TimeLoopLib import TreeDynamicTimeStepping
```
Comment (jbathmann) [inline MANGA.py:8]: I can only review this after merge of #99 and rebase

File: PopulationLib/GroupPlanting.py [hunk 2]
```diff
@@ -8,8 +8,7 @@
 if __name__ == '__main__' and __package__ is None:
     from os import sys, path
     sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
-import PopulationLib as PLib
-from PopulationLib import TreeGroup
+from pyMANGA.PopulationLib.TreeGroup import TreeGroup
```
Comment (jbathmann) [inline PopulationLib/GroupPlanting.py:11]: PLib is not required?

File: TreeModelLib/BelowgroundCompetition/SimpleTest/__init__.py [hunk 3]
```diff
@@ -6,4 +6,4 @@
 @author: bathmann
 """
 
-from .SimpleTest import SimpleTest
+from pyMANGA.TreeModelLib.BelowgroundCompetition.SimpleTest.SimpleTest import SimpleTest
```
Comment (jbathmann) [inline TreeModelLib/BelowgroundCompetition/SimpleTest/__init__.py:9]: Ok... I get the idea. When does this work know? Do I have to install MANGA via pip or can I - for development purposes - just append the path to the source code?

Imagine I got both i) the souce code and ii) the pip version. How to I define which version is used, if I type "import pyMANGA" somewhere on my system? Can you add a short explanation file for this?

**Review by jbathmann — CHANGES_REQUESTED** [review]
Some open questions in the comments... Otherwise thanks a lot!

File: TreeModelLib/BelowgroundCompetition/SimpleTest/__init__.py (hunk 3 above)
Comment (mcwimm) [reply TreeModelLib/BelowgroundCompetition/SimpleTest/__init__.py:9]: This is a problem at the moment. If pyManga is installed it uses the package and not the development source code....

**Comment by mcwimm** [comment]
outdated

**Pull request closed without merging**